draft-ietf-taps-impl.md

```diff
@@ -963,3 +963,46 @@ If the application indicates a bound on the expected Send or Receive bitrate, an
 
 * Cost Preferences:
 If the application indicates a preference to avoid expensive paths, and some paths are associated with a monetary cost, an implementation should decrease the ranking of such paths. If the application indicates that it prohibits using expensive paths, paths that are associated with a cost should be purged from the decision tree.
+
+# Reasons for errors {#appendix-reasons-errors}
+
+The Transport Services API {{I-D.ietf-taps-interface}} allows for the seven generic error types to specify a more detailed reason as to why an error occurred. This appendix lists some of the possible reasons.
+
+* BadConfiguration:
+The transport properties and endpoints provided by the application are either contradictory or incomplete. Examples include the lack of a remote endpoint on an active open or using a multicast group address while not requesting a unidirectional receive.
+
+* NoCandidates:
+None of the available transport protocols can satisfy the transport properties provided by the application.
+
+* Unresolvable:
+The remote or local specifier provided by the application can not be resolved.
```

Comment:
Maybe change this to "ResolutionFailed" or similar - "Unresolvable" sounds like an inherent and persistent problem with the specifier, but it might just be a problem with the resolver.

Reply:
+1